crates/gpui/src/platform/linux/platform.rs

```diff
@@ -390,10 +390,7 @@ impl Platform for LinuxPlatform {
             Ok(())
         })
     }
-
-    //todo!(linux): add trait methods for accessing the primary selection
```

Comment:
This one needs to be kept.

Reply:
Ah, I assumed it to be for the clipboard and moved it up :)

Reply:
Fixed